web/e2e/page-objects/steps/steps.ts

```diff
@@ -187,13 +187,19 @@ export class Steps extends AppPage {
     return element(by.id(`step-${option}-btn`));
   }
 
+  get stepDialogBox() {
+    return element(by.css("mat-dialog-container"));
```

Comment:
make sure it's an element or class?

Reply:
Both work "<mat-dialog-container aria-modal="true" class="mat-dialog-container ng-tns-c25-10 ng-trigger ng-trigger-dialogContainer ng-star-inserted" tabindex="-1""